### ISSUE TITLE
prometheus was collected before with the same name and label values

### ISSUE THREAD

**Issue by genofire** [mauricioabreu/nginx_rtmp_prometheus#5]
Errors if any stream is running:
```
An error has occurred while serving metrics:

25 error(s) occurred:
* collected metric "nginx_rtmp_stream_incoming_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_src" > counter:<value:1.8166564227e+10 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_outgoing_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_src" > counter:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_receive_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_src" > gauge:<value:4.8905792236328125 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_transmit_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_src" > gauge:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_uptime_seconds_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_src" > counter:<value:33318.732 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_incoming_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_480" > counter:<value:4.837126193e+09 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_outgoing_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_480" > counter:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_receive_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_480" > gauge:<value:1.454132080078125 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_transmit_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_480" > gauge:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_uptime_seconds_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_480" > counter:<value:33318.735 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_incoming_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_720" > counter:<value:8.594777839e+09 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_outgoing_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_720" > counter:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_receive_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_720" > gauge:<value:2.491180419921875 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_transmit_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_720" > gauge:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_uptime_seconds_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_720" > counter:<value:33318.735 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_incoming_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_1080" > counter:<value:1.7173994529e+10 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_outgoing_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_1080" > counter:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_receive_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_1080" > gauge:<value:4.895774841308594 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_transmit_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_1080" > gauge:<value:0 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_uptime_seconds_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8_1080" > counter:<value:33318.732 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_incoming_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8" > counter:<value:1.8167628543e+10 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_outgoing_bytes_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8" > counter:<value:1.8167628478e+10 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_receive_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8" > gauge:<value:4.874763488769531 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_transmit_bytes" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8" > gauge:<value:4.874763488769531 > } was collected before with the same name and label values
* collected metric "nginx_rtmp_stream_uptime_seconds_total" { label:<name:"stream" value:"c4eba06b-1ab3-4367-93e1-da584b96fcc8" > counter:<value:33320.122 > } was collected before with the same name and label values
```
---

maybe application label is missing

**Comment by genofire** [comment]
Stats-output of nginx

```xml
<?xml version="1.0" encoding="utf-8"?>
<?xml-stylesheet type="text/xsl" href="stat.xsl" ?>
<rtmp>
  <nginx_version>1.18.0</nginx_version>
  <nginx_rtmp_version>1.1.7.11-dev</nginx_rtmp_version>
  <built>Feb  6 2021 23:08:08</built>
  <pid>121996</pid>
  <uptime>117720</uptime>
  <naccepted>396</naccepted>
  <bw_in>39065168</bw_in>
  <bytes_in>459434742130</bytes_in>
  <bw_out>53640144</bw_out>
  <bytes_out>626821907492</bytes_out>
  <server>
    <application>
      <name>encoder</name>
      <live>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8</name>
          <time>33415242</time>
          <bw_in>4803320</bw_in>
          <bytes_in>18219551347</bytes_in>
          <bw_out>4803320</bw_out>
          <bytes_out>18218659533</bytes_out>
          <bw_audio>175024</bw_audio>
          <bw_video>4628296</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1492</id>
            <address>::ffff:127.0.0.1</address>
            <port>34294</port>
            <time>33415198</time>
            <flashver>LNX 9,0,124,2</flashver>
            <bytes_in>57005</bytes_in>
            <bytes_out>1061880361</bytes_out>
            <dropped>0</dropped>
            <avsync>-8</avsync>
            <timestamp>33416890</timestamp>
            <active/>
          </client>
          <client>
            <id>1491</id>
            <address>::ffff:127.0.0.1</address>
            <port>34292</port>
            <time>33415242</time>
            <flashver>LNX.11,1,102,55</flashver>
            <bytes_in>2136507917</bytes_in>
            <bytes_out>58713</bytes_out>
            <dropped>0</dropped>
            <avsync>-8</avsync>
            <timestamp>33416890</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1920</width>
              <height>1080</height>
              <frame_rate>25.000</frame_rate>
              <codec>H264</codec>
              <profile>High</profile>
              <compat>0</compat>
              <level>4.0</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>2</nclients>
          <publishing/>
          <active/>
        </stream>
        <nclients>2</nclients>
      </live>
    </application>
    <application>
      <name>out-mp4</name>
      <live>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_src</name>
          <time>33413855</time>
          <bw_in>4922160</bw_in>
          <bytes_in>18218470094</bytes_in>
          <bw_out>14766480</bw_out>
          <bytes_out>54649587995</bytes_out>
          <bw_audio>178832</bw_audio>
          <bw_video>4743328</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1511</id>
            <address>SECRET-EDGE_SERVER_1/show/</address>
            <port>0</port>
            <time>33413855</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>58745</bytes_in>
            <bytes_out>1061691033</bytes_out>
            <dropped>0</dropped>
            <avsync>32</avsync>
            <timestamp>33415296</timestamp>
            <active/>
          </client>
          <client>
            <id>1509</id>
            <address>127.0.0.1/hls/</address>
            <port>0</port>
            <time>33413855</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>58713</bytes_in>
            <bytes_out>1061691023</bytes_out>
            <dropped>0</dropped>
            <avsync>32</avsync>
            <timestamp>33415296</timestamp>
            <active/>
          </client>
          <client>
            <id>1510</id>
            <address>SECRET-EDGE_SERVER_2/show/</address>
            <port>0</port>
            <time>33413855</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>58729</bytes_in>
            <bytes_out>1055862733</bytes_out>
            <dropped>87</dropped>
            <avsync>32</avsync>
            <timestamp>33415296</timestamp>
            <active/>
          </client>
          <client>
            <id>1508</id>
            <address>::ffff:127.0.0.1</address>
            <port>34332</port>
            <time>33413855</time>
            <flashver>FMLE/3.0 (compatible; Lavf58.45</flashver>
            <bytes_in>2135287960</bytes_in>
            <bytes_out>553</bytes_out>
            <dropped>0</dropped>
            <avsync>32</avsync>
            <timestamp>33415296</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1920</width>
              <height>1080</height>
              <frame_rate>25.000</frame_rate>
              <codec>H264</codec>
              <profile>High</profile>
              <compat>0</compat>
              <level>4.0</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>4</nclients>
          <publishing/>
          <active/>
        </stream>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_480</name>
          <time>33413862</time>
          <bw_in>1432328</bw_in>
          <bytes_in>4852317966</bytes_in>
          <bw_out>4296984</bw_out>
          <bytes_out>14555124948</bytes_out>
          <bw_audio>107792</bw_audio>
          <bw_video>1324528</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1496</id>
            <address>SECRET-EDGE_SERVER_1/show/</address>
            <port>0</port>
            <time>33413862</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>15993</bytes_in>
            <bytes_out>578603653</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1495</id>
            <address>SECRET-EDGE_SERVER_2/show/</address>
            <port>0</port>
            <time>33413862</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>15977</bytes_in>
            <bytes_out>576768584</bytes_out>
            <dropped>95</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1494</id>
            <address>127.0.0.1/hls/</address>
            <port>0</port>
            <time>33413862</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>15993</bytes_in>
            <bytes_out>578603643</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1493</id>
            <address>::ffff:127.0.0.1</address>
            <port>34308</port>
            <time>33413862</time>
            <flashver>FMLE/3.0 (compatible; Lavf58.45</flashver>
            <bytes_in>846871334</bytes_in>
            <bytes_out>553</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>854</width>
              <height>480</height>
              <frame_rate>30.000</frame_rate>
              <codec>H264</codec>
              <profile>Baseline</profile>
              <compat>192</compat>
              <level>3.1</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>4</nclients>
          <publishing/>
          <active/>
        </stream>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_720</name>
          <time>33413858</time>
          <bw_in>2465080</bw_in>
          <bytes_in>8620980621</bytes_in>
          <bw_out>7395248</bw_out>
          <bytes_out>25860070606</bytes_out>
          <bw_audio>143408</bw_audio>
          <bw_video>2321672</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1501</id>
            <address>SECRET-EDGE_SERVER_1/show/</address>
            <port>0</port>
            <time>33413858</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>28041</bytes_in>
            <bytes_out>53103027</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1500</id>
            <address>SECRET-EDGE_SERVER_2/show/</address>
            <port>0</port>
            <time>33413858</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>28041</bytes_in>
            <bytes_out>50225954</bytes_out>
            <dropped>95</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1499</id>
            <address>127.0.0.1/hls/</address>
            <port>0</port>
            <time>33413858</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>28041</bytes_in>
            <bytes_out>53103017</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1498</id>
            <address>::ffff:127.0.0.1</address>
            <port>34316</port>
            <time>33413858</time>
            <flashver>FMLE/3.0 (compatible; Lavf58.45</flashver>
            <bytes_in>589831234</bytes_in>
            <bytes_out>553</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1280</width>
              <height>720</height>
              <frame_rate>30.000</frame_rate>
              <codec>H264</codec>
              <profile>Baseline</profile>
              <compat>192</compat>
              <level>3.1</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>4</nclients>
          <publishing/>
          <active/>
        </stream>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_1080</name>
          <time>33413858</time>
          <bw_in>4859056</bw_in>
          <bytes_in>17225618516</bytes_in>
          <bw_out>14577168</bw_out>
          <bytes_out>51670410311</bytes_out>
          <bw_audio>216296</bw_audio>
          <bw_video>4642752</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1506</id>
            <address>SECRET-EDGE_SERVER_1/show/</address>
            <port>0</port>
            <time>33413858</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>55577</bytes_in>
            <bytes_out>69811587</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1504</id>
            <address>127.0.0.1/hls/</address>
            <port>0</port>
            <time>33413858</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>55529</bytes_in>
            <bytes_out>69811577</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1505</id>
            <address>SECRET-EDGE_SERVER_2/show/</address>
            <port>0</port>
            <time>33413858</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>55561</bytes_in>
            <bytes_out>63358700</bytes_out>
            <dropped>93</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <active/>
          </client>
          <client>
            <id>1503</id>
            <address>::ffff:127.0.0.1</address>
            <port>34324</port>
            <time>33413858</time>
            <flashver>FMLE/3.0 (compatible; Lavf58.45</flashver>
            <bytes_in>1143436242</bytes_in>
            <bytes_out>554</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1920</width>
              <height>1080</height>
              <frame_rate>30.000</frame_rate>
              <codec>H264</codec>
              <profile>Baseline</profile>
              <compat>192</compat>
              <level>4.0</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>4</nclients>
          <publishing/>
          <active/>
        </stream>
        <nclients>16</nclients>
      </live>
    </application>
    <application>
      <name>hls</name>
      <live>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_src</name>
          <time>33413855</time>
          <bw_in>4922160</bw_in>
          <bytes_in>18218470094</bytes_in>
          <bw_out>0</bw_out>
          <bytes_out>0</bytes_out>
          <bw_audio>178832</bw_audio>
          <bw_video>4743328</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1512</id>
            <address>::ffff:127.0.0.1</address>
            <port>34334</port>
            <time>33413855</time>
            <flashver>LNX.11,1,102,55</flashver>
            <bytes_in>2135425790</bytes_in>
            <bytes_out>58713</bytes_out>
            <dropped>0</dropped>
            <avsync>32</avsync>
            <timestamp>33415296</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1920</width>
              <height>1080</height>
              <frame_rate>25.000</frame_rate>
              <codec>H264</codec>
              <profile>High</profile>
              <compat>0</compat>
              <level>4.0</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>1</nclients>
          <publishing/>
          <active/>
        </stream>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_480</name>
          <time>33413858</time>
          <bw_in>1432328</bw_in>
          <bytes_in>4852317966</bytes_in>
          <bw_out>0</bw_out>
          <bytes_out>0</bytes_out>
          <bw_audio>107792</bw_audio>
          <bw_video>1324528</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1497</id>
            <address>::ffff:127.0.0.1</address>
            <port>34310</port>
            <time>33413858</time>
            <flashver>LNX.11,1,102,55</flashver>
            <bytes_in>847038321</bytes_in>
            <bytes_out>15993</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>854</width>
              <height>480</height>
              <frame_rate>30.000</frame_rate>
              <codec>H264</codec>
              <profile>Baseline</profile>
              <compat>192</compat>
              <level>3.1</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>1</nclients>
          <publishing/>
          <active/>
        </stream>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_720</name>
          <time>33413858</time>
          <bw_in>2465080</bw_in>
          <bytes_in>8620980621</bytes_in>
          <bw_out>0</bw_out>
          <bytes_out>0</bytes_out>
          <bw_audio>143408</bw_audio>
          <bw_video>2321672</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1502</id>
            <address>::ffff:127.0.0.1</address>
            <port>34318</port>
            <time>33413858</time>
            <flashver>LNX.11,1,102,55</flashver>
            <bytes_in>589970253</bytes_in>
            <bytes_out>28041</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1280</width>
              <height>720</height>
              <frame_rate>30.000</frame_rate>
              <codec>H264</codec>
              <profile>Baseline</profile>
              <compat>192</compat>
              <level>3.1</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>1</nclients>
          <publishing/>
          <active/>
        </stream>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8_1080</name>
          <time>33413855</time>
          <bw_in>4859056</bw_in>
          <bytes_in>17225618516</bytes_in>
          <bw_out>0</bw_out>
          <bytes_out>0</bytes_out>
          <bw_audio>216296</bw_audio>
          <bw_video>4642752</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1507</id>
            <address>::ffff:127.0.0.1</address>
            <port>34326</port>
            <time>33413855</time>
            <flashver>LNX.11,1,102,55</flashver>
            <bytes_in>1143543947</bytes_in>
            <bytes_out>55529</bytes_out>
            <dropped>0</dropped>
            <avsync>-3</avsync>
            <timestamp>33415021</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1920</width>
              <height>1080</height>
              <frame_rate>30.000</frame_rate>
              <codec>H264</codec>
              <profile>Baseline</profile>
              <compat>192</compat>
              <level>4.0</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>1</nclients>
          <publishing/>
          <active/>
        </stream>
        <nclients>4</nclients>
      </live>
    </application>
    <application>
      <name>stream</name>
      <live>
        <stream>
          <name>c4eba06b-1ab3-4367-93e1-da584b96fcc8</name>
          <time>33415245</time>
          <bw_in>4803320</bw_in>
          <bytes_in>18219551347</bytes_in>
          <bw_out>4803320</bw_out>
          <bytes_out>18219551282</bytes_out>
          <bw_audio>175024</bw_audio>
          <bw_video>4628296</bw_video>
          <bw_data>0</bw_data>
          <client>
            <id>1490</id>
            <address>127.0.0.1/encoder</address>
            <port>0</port>
            <time>33415245</time>
            <flashver>ngx-local-relay</flashver>
            <bytes_in>58713</bytes_in>
            <bytes_out>1062773509</bytes_out>
            <dropped>0</dropped>
            <avsync>-8</avsync>
            <timestamp>33416890</timestamp>
            <active/>
          </client>
          <client>
            <id>1489</id>
            <address>fd42:ffc:ab1e::333:1</address>
            <port>50002</port>
            <time>33415305</time>
            <flashver>FMLE/3.0 (compatible; Lavf58.29</flashver>
            <bytes_in>2136370581</bytes_in>
            <bytes_out>549</bytes_out>
            <dropped>0</dropped>
            <avsync>-8</avsync>
            <timestamp>33416890</timestamp>
            <publishing/>
            <active/>
          </client>
          <meta>
            <video>
              <width>1920</width>
              <height>1080</height>
              <frame_rate>25.000</frame_rate>
              <codec>H264</codec>
              <profile>High</profile>
              <compat>0</compat>
              <level>4.0</level>
            </video>
            <audio>
              <codec>AAC</codec>
              <profile>LC</profile>
              <channels>2</channels>
              <sample_rate>48000</sample_rate>
            </audio>
          </meta>
          <nclients>2</nclients>
          <publishing/>
          <active/>
        </stream>
        <nclients>2</nclients>
      </live>
    </application>
  </server>
</rtmp>
```

**Comment by genofire** [comment]
fixed in 7335f7d7e824b49b113b33978e172a02c938c319 - maybe "we" should release v0.2.1

**Comment by mauricioabreu** [comment]
@genofire I just pushed a new version. Can you test latest/master/v2.0.1?
